sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -168,7 +171,7 @@ internal sealed class ServiceBusListener : IListener, IScaleMonitorProvider, ITa
 
             if (_supportMinBatchSize)
             {
-                _cachedMessagesManager = new(options.MaxMessageBatchSize, options.MinMessageBatchSize);
+                _cachedMessagesManager = new(_maxMessageBatchSize, options.MinMessageBatchSize);
```

Comment:
I think we will still need to support setting max message batch size from the host.json, but just prefer the value set in the trigger.

i.e.:

```suggestion
                _cachedMessagesManager = new(_maxMessageBatchSize > 0 ? _maxMessageBatchSize : options.MaxMessageBatchSize, options.MinMessageBatchSize);
```

Reply:
Nevermind I see you're already handling that upstream - my bad!